src/Castle.Windsor.Extensions.DependencyInjection/Scope/ExtensionContainerScopeAccessor.cs

```diff
@@ -23,11 +23,7 @@ internal class ExtensionContainerScopeAccessor : IScopeAccessor
 	{
 		public ILifetimeScope GetScope(CreationContext context)
 		{
-			if(ExtensionContainerScope.Current == null)
-			{
-				throw new InvalidOperationException("No scope available");
-			}
-			return ExtensionContainerScope.Current;
+			return ExtensionContainerScope.Current ??  throw new InvalidOperationException("No scope available");
```

Comment:
Double space

Haha, we've got an `InvalidOperationException` here for no scope available. Maybe should just have a static `CurrentOrThrow` property both can use?

Reply:
I can change, but may effect this in a funny way:
![image](https://user-images.githubusercontent.com/2411563/101637664-58a3c800-3a2d-11eb-8b5c-eb02acede83a.png)

Reply:
Ah yer, probably easiest to just duplicate the throw statement for now.

Reply:
I thnk the CurrentOrThrow is best, then you know exactly what is null. No Current, that will throw, no root, then this will throw. Please see update (still not my PR, i cannot resolve :-/ )